app/components/sidebars/settings/settings_sidebar.js

```diff
@@ -293,6 +304,15 @@ export default class SettingsDrawer extends PureComponent {
                         </View>
                         <View style={style.separator}/>
                         <View style={style.block}>
+                            <DrawerItem
+                                defaultMessage='Edit Profile'
+                                i18nId='mobile.routes.edit_profile'
+                                iconName='ios-person'
+                                iconType='ion'
+                                onPress={this.goToEditProfile}
+                                separator={true}
+                                theme={theme}
+                            />
```

Comment:
Please add unit test related to change - test if drawer item is rendered correctly and test functionality of `goToUserProfile`.